modules/javafx.controls/src/main/java/javafx/scene/control/SkinBase.java

```diff
@@ -206,29 +209,31 @@ protected final void consumeMouseEvents(boolean value) {
 
 
     /**
-     * Subclasses can invoke this method to register that they want to listen to
-     * property change events for the given property. Registered {@link Consumer} instances
-     * will be executed in the order in which they are registered.
-     * @param property the property
-     * @param consumer the consumer
+     * Registers an operation to perform when the given {@code property} sends a change event.
+     * Does nothing if either {@code property} or {@code operation} are {@code null}.
+     * If multiple operations are registered on the same property, they will be performed in the
+     * order in which they were registered.
+     *
+     * @param property the property to observe for change events, may be {@code null}
+     * @param operation the operation to perform when the property sends a change event,
+     *  may be {@code null}
      */
-    protected final void registerChangeListener(ObservableValue<?> property, Consumer<ObservableValue<?>> consumer) {
+    protected final void registerChangeListener(ObservableValue<?> property, Consumer<ObservableValue<?>> operation) {
```

Comment:
Maybe instead of `property` use `observable` like with the invalidation methods to not confuse it with the `Property` interface.

Reply:
changed as suggested - hope everybody is happy now :)